### ISSUE TITLE
Question receiving ETN

### ISSUE THREAD

**Issue by tcoined** [tperalta82/electroneum-core#12]
I just started mining ent and was able to mine up to my first payment of a whopping 11 etn before the blockchain synchronized in the new install of the gui wallet.  I figured the balance would update once the wallet deamon was up to date but had no such luck... I can see the transaction was confirmed in the blockchain explorer but have no reference to it in the wallet. Are those coins burned because I hadn't synchronized yet? 
![etn-gui-wallet-20180218](https://user-images.githubusercontent.com/36612471/36359785-30117a22-14ec-11e8-9f9e-c52f9a89e853.JPG)

**Comment by tperalta82** [comment]
This wallet is deprecated, use their official wallet with your private keys to import it there!

**Comment by tcoined** [comment]
Ok , so I wont us this wallet in the future but do have one more quick question... It was seemingly easy  to find the keys in the gui wallet, import them into  the web/mobile wallet which I had already signed up for but now that I have transferred the addresses will that receive address now be registered with the new wallet? (i.e. do I need to reset my miner and reimport keys after my next pool balance is sent?) Thanks for helping me out only been invested in crypto for a couple months its exciting but there is a lot to learn about a lot of different things. Your help is appreciated.  



    On Sunday, February 18, 2018 8:44 PM, Tiago Peralta <notifications@github.com> wrote:
 

 This wallet is deprecated, use their official wallet with your private keys to import it there!—
